### PR TITLE
Add related projects as a community extension

### PR DESCRIPTION
**Pull request by transpresupuestaria**

### CODE REVIEW

**Comment by timgdavies** [comment]
Just a quick note to let you know we're aware of this request - and will review over the next week.

File: related_projects/entry.json [hunk 1]
```diff
@@ -0,0 +1,14 @@
+[{
+  "name": {
+    "en" : "Related projects"
+  },
+  "description": {
+    "en": "Agregate to the community extensions in order to directly link the contracts with their respective projects (when applies)"
+  },
```
Comment (julijahansen) [inline related_projects/entry.json]: Would suggest to update the description to: "Adds a related projects array to allow for the contracting process to be linked to its respective projects."

File: related_projects/entry.json [hunk 2]
```diff
@@ -0,0 +1,14 @@
+[{
+  "name": {
+    "en" : "Related projects"
+  },
```
Comment (julijahansen) [inline related_projects/entry.json]: Removed underscore from the name.

**Comment by jpmckinney** [comment]
@transpresupuestaria Apologies for the delay in the review. To be considered for inclusion in the extension registry, the extension should be publisher-agnostic. In other words, references to SHCP should be removed from the name and description in extension.json. You can of course include information about the extension's author in the readme.

The readme also includes some unusual content, e.g. a discussion of item classification schemes, as part of the documentation for project sector and project classifications. The simplest solution would be to remove these extra field descriptions, as they are already described as part of the [Budget and Projects](https://extensions.open-contracting.org/en/extensions/budget_project/master/schema/#project) extension, which defines the `Project` definition used here.

With those two changes, we can add this to the registry.

cc @yolile

**Comment by jpmckinney** [comment]
I performed a more technical review and identified further issues:

* Trailing comma, causing invalid JSON https://github.com/transpresupuestaria/ocds_related_projects_extension/pull/1/files
* `relatedProjects` is an array, but there is no `items` property; instead `properties` is given, but this is incorrect. Same with `locations` and `additionalClassifications`.
* `relatedProjects` should re-use the `Project` definition in the ocds_budget_projects_extension extension.

I've proposed some changes in https://github.com/transpresupuestaria/ocds_related_projects_extension/pull/3 but my earlier comment would still need to be resolved.

I'll close this issue until those PRs are merged and other issues fixed.

**Pull request closed without merging**